asgi.py

```
@@ -0,0 +1,34 @@
+# Export the ASGI application
+application = app
+
+if __name__ == "__main__":
+    import uvicorn
+    
+    # Enable reload in development mode
+    is_debug = os.getenv("BACKEND_BUILD_TARGET") == "debug"
+    
+    uvicorn.run(
+        "asgi:application",
+        host="0.0.0.0",
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)